python/benchmarks/spotify/datagen.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#
+
+import os
+from random import choice
+
+import lance
+import pyarrow as pa
+import pandas as pd
+from gensim.models.word2vec import Word2Vec
+
+
+def generate_embeddings(col: pd.Series, **hyper_params) -> pd.Series:
+    # unique_tracks
+    sentences = []
+    for s in col:
+        if not isinstance(s, str):
+            continue
+        # We could prob do more cleaning here
+        sentences.append(s)
+    model = Word2Vec(sentences, **hyper_params)
+    print(f"Vector space size: {len(model.wv.index_to_key)}")
+    embeddings = []
+    for query_item in col:
+        if query_item not in model.wv:
+            query_item = choice(list(model.wv.index_to_key))
+        embeding = model.wv[query_item]
+        embeddings.append(embeding)
+
+    print(embeddings)
+    return embeddings
+
+
+def main():
+    df = pd.read_csv("spotify_dataset.csv", on_bad_lines="skip")
```

Comment:
add a comment to indicate where to get the data?

Reply:
done.